chore(docs): Add paths to test schema files (#1564)
cerbos/cerbos: docs/modules/policies/pages/authoring_tips.adoc

include::ROOT:partial$attributes.adoc[]

:policy-schema-file: /policy/v1/Policy.schema.json
:test-schema-file: /policy/v1/TestSuite.schema.json
:current-policy-schema-url: {app-schema-url-current}{policy-schema-file}
:latest-policy-schema-url: {app-schema-url-latest}{policy-schema-file}
:current-test-schema-url: {app-schema-url-current}{test-schema-file}
:latest-test-schema-url: {app-schema-url-latest}{test-schema-file}

= Policy authoring 

== Tips for working with policies

* Policies can be in either YAML or JSON formats. Accepted file extensions are `.yml`, `.yaml` or `.json`. All other extensions are ignored.

> * The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and xref:compile.adoc[tests] at `{current-test-schema-url}`. If you prefer to always use the latest version, it can be accessed at `{latest-policy-schema-url}` and `{latst-test-schema-url}`. 

Reviewer: ```suggestion
* The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and the xref:compile.adoc[test suite] schema is available at `{current-test-schema-url}`. If you prefer to always use the latest version, they can be accessed at `{latest-policy-schema-url}` and `{latst-test-schema-url}` respectively. 
```

Reviewer 2: `{latst-test-schema-url}` -> ``{latest-test-schema-url}``

[source,yaml,subs="+attributes"]
----
yaml.schemas: {
    "{latest-policy-schema-url}": "/cerbos/*",

>     "{latest-test-schema-url}": "/cerbos/tests/*",

Reviewer: ```suggestion
    "{latest-test-schema-url}": "/cerbos/**/*_test.yaml",
```

            settings = {
                yaml = {
                    schemas = {
                        "{latest-policy-schema-url}": "/cerbos/*",

>                         "{latest-test-schema-url}": "/cerbos/tests/*"

Reviewer: ```suggestion
                        "{latest-test-schema-url}": "/cerbos/**/*_test.yaml"
```

----
{
  "yaml.schemas": {
    "{latest-policy-schema-url}": "cerbos/*",

>     "{latest-test-schema-url}": "/cerbos/tests/*",

Reviewer: ```suggestion
    "{latest-test-schema-url}": "/cerbos/**/*_test.yaml",
```
